plugins/wasd_controls.js

```diff
@@ -86,14 +93,21 @@ BBPlugin.register('wasd_controls', {
 			type: 'number',
 			value: 100,
 			min: 1
-		}))
+		}));
 
 		deletables.push(new Setting('wasd_y_level', {
 			name: 'WASD Navigation at Y Level',
 			description: 'Navigate using WASD at consistent Y level rather than on camera plane',
 			category: 'preview',
 			value: true
-		}))
+		}));
+
+		deletables.push(new Setting('requires_hold_right_mouse', {
+			name: 'Only works when holding the right mouse button',
+			description: 'The WASD Controls needs to be enabled for this to work.',
+			category: 'preview',
+			value: true
```

Comment:
Let's make this false by default to not break the workflow for people who are used to how it works now.